spec/requests/datacite_dois_spec.rb

```diff
@@ -4113,9 +4113,10 @@
     # end
   end
 
-  describe "GET /dois/get-dois", vcr: true do
-    let(:prefix) { create(:prefix, uid: "10.5438") }
-    let!(:client_prefix) { create(:client_prefix, prefix: prefix, client: client) }
+  describe "GET /dois/get-dois", :skip_prefix_pool, vcr: true do
```

Comment:
Instead of needing to skip prefix pool, cant the test be changed to just use one of the ones created?

Reply:
I am getting rid of any other ':skip_prefix_pool's that I can find.